src/reaper_osara.cpp

```diff
@@ -3762,6 +3776,57 @@ void cmdToggleLoopSegScrub(Command* command) {
 	Main_OnCommand(command->gaccel.accel.cmd, 0);
 }
 
+void cmdReportRegionMarkerItems(Command* command) {
+	const bool multiLine = lastCommandRepeatCount == 1;
+	ostringstream s;
+	auto separate = [&s, multiLine]() {
+		if(s.tellp()) {
+			s << (multiLine ? "\r\n" : ", ");
+		}
+	};
+	double pos = GetPlayState()? GetPlayPosition():GetCursorPosition();
+	double start,end;
+	bool isrgn;
+	int number;
+	const char* name{nullptr};
+	int idx = 0;
+	while(EnumProjectMarkers(idx++, &isrgn, &start, &end, &name, &number)>0) {
+		if(!(isrgn && start <= pos && pos <= end)) {
+			continue;
+		} 
+		separate();
+		if(name[0]) {
+			s << name ;
+		} else {
+			// Translators: used to report an unnamed region. {} is replaced with the region number.  
+			s << format(translate("region {}"), number);
+		}
+	}
+	int markerIdx;
+	GetLastMarkerAndCurRegion(nullptr, pos, &markerIdx, nullptr);
+	if(markerIdx >=0) {
+		EnumProjectMarkers(markerIdx, nullptr, nullptr, nullptr, &name, &number);
+		separate();
+		if (name[0]) {
+			s << name;
+		} else {
+			// Translators: used to report an unnamed marker. {} is replaced with the marker number.  
+			s << format(translate("marker {}"), number);
+		}
+	}
+	separate();
+	s << formatItemsWithState([pos](MediaItem* item) -> bool{
+		MediaTrack* track = GetMediaItem_Track(item);
+		return (isTrackSelected(track) && isPosInItem(pos, item));
+	}, multiLine);
+	if(multiLine) {
+		// Translators: The title of the review message for the action "OSARA: report regions, most recent marker and items at current position".
+		reviewMessage(translate("regions, most recent marker and items at current position"), s.str().c_str());
```

Comment:
1. nit: GUI is the one case where we don't use all lower case. For controls, sentence case; i.e. only upper case the first letter. For dialog titles, title case.
2. This is a very verbose title. Could we shorten it to "At Current Position" or something?